cms_index.php

```diff
@@ -0,0 +1,109 @@
+<?php
+require_once 'php/dbConnection.php';
+require_once 'php/functions.php';
+
+$db = getdbConnection();
+
+if (isset($_POST['addAboutMeText'])) {
+    $addAboutMeText = $_POST['addAboutMeText'];
+    $cleanAboutMeText = removeWhitespaceHTML($addAboutMeText);
+    $checkAddMeText = checkTextExists($cleanAboutMeText);
+    $newAboutMeText = addAboutMeText($db, $checkAddMeText, $cleanAboutMeText);
+    $addAboutMeSuccess = aboutMeSuccess($newAboutMeText);
+}
+
+if(isset($_POST['selectAboutMeText'])) {
+    $editId = $_POST['selectAboutMeText'];
+    $aboutTextToEdit = getAboutTextToEdit($db, $editId);
+    $displayAboutTextToEdit = displayAboutTextToEdit($aboutTextToEdit);
+    $displaySubmitEditButton = displaySubmitEditButton();
+}
+
+if(isset($_POST['editAboutMeTextAndQuote'])) {
+    $submitEditText = $_POST['editAboutMeTextAndQuote'];
+    $cleanEditText = removeWhitespaceHTML($submitEditText);
+    $checkEditText = checkTextExists($cleanEditText);
+    $editId = $_POST['editId'];
+    $updateAboutMeQuoteAndText = updateAboutMeQuoteAndText($db, $checkEditText, $cleanEditText, $editId);
+    $editSuccessMessage = aboutMeSuccess($updateAboutMeQuoteAndText);
+}
+
+if(isset($_POST['deleteAboutMeText'])) {
+    $deleteAboutMePara = $_POST['deleteAboutMeText'];
+    $aboutMeTextDeleted = deleteAboutMeText($db, $deleteAboutMePara);
+    $deleteSuccessMessage = deleteAboutMeSuccess($aboutMeTextDeleted);
+}
+
+$aboutMeTextAndQuote = getAboutMeTextAndQuote($db);
+$displayAboutMeDropdown = aboutMeTextDropdown($aboutMeTextAndQuote);
+
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Admin for Kate's Portfolio Page</title>
+    <link rel='stylesheet' type='text/css' href='css/normalize.css'/>
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:300,500" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' href='css/cms_styles.css'/>
+</head>
+<body>
+<h1>Content Managing Kate's Portfolio Page</h1>
+<section class="aboutMe">
+    <h2>About Me</h2>
+    <form method="POST" action="" id="addAboutMeText">
```

Comment:
change action to cms_index.php so it looks intentional that you are submitting to the same page

Reply:
Done